pom.xml

```diff
@@ -19,6 +19,11 @@
     <url>http://www.jooq.org</url>
 
     <properties>
+        <!--
+        File encodings.
+        -->
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
```

Comment:
This change seems unrelated to the actual set of changes

Reply:
Jooq emits warnings about encoding not set, this makes them go away